host/4/bookworm/java/java21/java21-appservice.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG JAVA_VERSION=21.0.1
 ARG JAVA_HOME=/usr/lib/jvm/msft-21-x64
 # Build the runtime from source
-ARG HOST_VERSION=4.34.2
+ARG HOST_VERSION=4.1034.2
```

Comment:
FYI - I will share the new tag EOD. 4.1035.0

Reply:
done updated